### ISSUE TITLE
genutils is used but not specified as a requirement

### ISSUE THREAD

**Issue by frenzymadness** [takluyver/jupyter_protocol#6]
See: https://github.com/takluyver/jupyter_protocol/search?q=genutils

**Comment by takluyver** [comment]
You're quite right, but if you're looking at this, be aware that it's essentially a dead repo at this point. It was part of a big refactoring of how kernels are launched and used that I was working on but didn't go anywhere.

**Comment by frenzymadness** [comment]
I know, but we have this is Fedora as an RPM so I wanted to just open an issue here. Feel free to ignore it. Or better might be to archive the repository?

**Comment by takluyver** [comment]
I might archive it, though I'm still speculating about creating some sort of refreshed API for working with kernels, which might use this as a starting point.

Is there anything in Fedora that depends on this? If not, you might want to consider removing it.

**Comment by apteryks** [comment]
In GNU Guix, there's a package for https://github.com/takluyver/jupyter_kernel_mgmt which uses it (another one of your projects :-)).
